core/DataTable.php

```diff
@@ -1460,12 +1459,13 @@ public function addRowsFromSimpleArray($array)
                 // it cannot be lost during the conversion. Because we are not able to handle properly
                 // this key, we throw an explicit exception.
                 if (is_string($key)) {
-                    throw $e;
+                    // we define an exception we may throw if at one point we notice that we cannot handle the data structure
+                    throw new Exception(sprintf($exceptionText, var_export($array, true)));
```

Comment:
Only call `var_export` when needed... we called it > 100k times although it was not needed